samples/client/petstore/go/go-petstore/api_store.go

```diff
@@ -144,7 +144,6 @@ func (a *StoreApiService) GetInventory(ctx context.Context) (map[string]int32, *
 			localVarHeaderParams["api_key"] = key
 		}
 	}
-
```

Comment:
Why were these files touched?

Reply:
I changed the go files since otherwise the generated code did not match the samples and seemed to fail the CI/CD checks. I noted in the commit comments - I really have no idea why - it looks like a newline at the end of a file. Usually a generated on Windows vs Unix thing.